ddtrace/contrib/_logbook.py

```diff
@@ -16,7 +16,8 @@
 Patch ``logbook``
 ~~~~~~~~~~~~~~~~~~~
 
-If using :ref:`ddtrace-run<ddtracerun>` then set the environment variable ``DD_LOGS_INJECTION=true``.
+Logbook support is auto-enabled when :ref:`ddtrace-run<ddtracerun>` is used.
```

Comment:
We should specify this is only for structured logs since not everyone will correlate this back to the release notes.